pkg/gofr/datasource/clickhouse/clickhouse_test.go

```diff
@@ -91,8 +92,9 @@ func Test_ClickHouse_HealthDOWN(t *testing.T) {
 
 	mockConn.EXPECT().Ping(gomock.Any()).Return(sql.ErrConnDone)
 
-	resp := c.HealthCheck()
+	resp, err := c.HealthCheck(context.Background())
 
+	assert.Equal(t, err, errors.New(errStatusDown))
```

Comment:
```suggestion
	assert.ErrorIs(t, err, ErrStatusDown))
```

See https://github.com/gofr-dev/gofr/pull/780#discussion_r1666685504